src/core/agent/services/signifyNotificationService.ts

```diff
@@ -606,6 +644,21 @@ class SignifyNotificationService extends AgentService {
             .get(admitExchange.exn.p);
           const credentialId = grantExchange.exn.e.acdc.d;
           if (credentialId) {
+            const holder = await this.identifierStorage.getIdentifierMetadata(
+              admitExchange.exn.i
+            );
+            if (holder.multisigManageAid) {
+              const notifications =
+                  await this.notificationStorage.findAllByQuery({
+                    exnSaid: grantExchange.exn.d,
+                  });
+              for (const notification of notifications) {
+                this.deleteNotificationRecordById(
```

Comment:
Can we add a TODO line to delete other long running operations in `linkedGroupRequests` - once my PR for KERIA is merged we will have different operation IDs so there can be multiple